.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+#### What does this PR do?
+
+[GPII-XXXX](https://issues.gpii.net/browse/GPII-XXXX)
+
+##### Description of the fix implemented
+
+##### Does this PR affect to the living environments? if so, how?
+
+<!--
+The following sections are optional, remove them if needed
+-->
+
+##### Notes for the reviewers
+
+##### Deployment plan
+
+ - [ ] Step 1
+ - [ ] Step 2
+
+##### PR dependencies
+
+ - https://github.com/gpii-ops/gpii-infra/pull/1
+
+##### Tests done by the creator
```

Comment:
Perhaps _author_, or just _"Tests performed"_. I'm not quite sure about this section - being this optional kind of suggests we don't expect authors to do any tests? :) But then I don't have a better solution.